runtime-common/stdlib/serialization/serialization-context.h

```diff
@@ -13,6 +16,16 @@ struct SerializationLibContext final : private vk::not_copyable {
   // msgpack serialization
   size_t instance_depth{0};
   uint32_t serialize_as_float32_{0};
+  std::optional<std::array<char, 256>> msgpack_error{std::nullopt};
```

Comment:
Let's use constant instead of hardcoded value. Also, it's better to just use default init here `msgpack_error{}`

Reply:
`msgpack_error{}` is forced by formatter to be `msgpack_error;`. I would prefer explicit init here

Reply:
Then it's better to keep it as `msgpack_error;`. It's common approach for our source code

Reply:
Done